src/extension.ts

```diff
@@ -391,16 +392,16 @@ export class Extension implements RunHooks {
     }
   }
 
-  private async _runTests(include: readonly vscodeTypes.TestItem[] | undefined, mode: 'run' | 'debug' | 'watch') {
+  private async _runTests(request: vscodeTypes.TestRunRequest, mode: 'run' | 'debug' | 'watch') {
     this._completedSteps.clear();
     this._executionLinesChanged();
+    const include = request.include;
 
     // Create a test run that potentially includes all the test items.
     // This allows running setup tests that are outside of the scope of the
     // selected test items.
     const rootItems: vscodeTypes.TestItem[] = [];
```

Comment:
This can be removed since https://github.com/microsoft/vscode/issues/213182 has been landed upstream.

Reply:
You mean remove the comment? I think the code is used.